package.json

```diff
@@ -86,42 +80,38 @@
     "datatables.net-responsive-bs4": "^2.2.9",
     "datatables.net-select-bs4": "^1.3.1",
     "ekko-lightbox": "~5.3.0",
-    "fibers": "^4.0.3",
-    "flatpickr": "^4.6.3",
-    "font-awesome": "~4.7.0",
+    "flatpickr": "^4.6.13",
+    "font-awesome": "^4.7.0",
     "form-serializer": "~2.5.0",
-    "globalthis": "^1.0.3",
-    "handlebars": "^4.7.7",
-    "hls.js": "^1.4.10",
-    "html-to-image": "^1.7.0",
-    "imagesloaded": "^4.1.4",
-    "jquery": "^3.5.1",
-    "jquery-ui": "~1.13.2",
-    "jquery-validation": "~1.19.5",
-    "jstree": "^3.3.10",
-    "leaflet": "^1.7.1",
+    "globalthis": "^1.0.4",
+    "handlebars": "^4.7.8",
+    "hls.js": "^1.5.13",
+    "html-to-image": "^1.11.11",
+    "imagesloaded": "^5.0.0",
+    "jquery": "^3.7.1",
+    "jquery-ui": "~1.13.3",
+    "jquery-validation": "~1.21.0",
+    "jstree": "^3.3.16",
+    "leaflet": "^1.9.4",
     "leaflet-draw": "^1.0.4",
     "leaflet-easyprint": "^2.1.9",
     "leaflet-fullscreen": "^1.0.2",
-    "leaflet-search": "^3.0.3",
-    "leaflet.markercluster": "^1.4.1",
+    "leaflet-search": "^4.0.0",
+    "leaflet.markercluster": "^1.5.3",
     "masonry-layout": "^4.2.2",
-    "material-design-icons": "^3.0.1",
-    "moment": "~2.29.4",
-    "moment-jalaali": "~0.7.2",
-    "moment-timezone": "^0.5.35",
-    "monaco-editor": "^0.46.0",
-    "moveable": "^0.48.0",
-    "pdfjs-dist": "2.4.456",
-    "persian-date": "^0.3.1-b",
+    "moment": "~2.30.1",
+    "moment-jalaali": "~0.10.1",
+    "moment-timezone": "^0.5.45",
+    "monaco-editor": "^0.50.0",
+    "moveable": "^0.53.0",
+    "pdfjs-dist": "^4.5.136",
```

Comment:
When we looked at a v4 upgrade before we ran into errors because the accompanying asset wasn't the same `/modules/assets/pdfjs/pdf.worker.js`. Have you managed to cater for that?